portal/apps/org.wso2.carbon.iam.admin-portal.app/src/main/fragments/ask-password/ask-password.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="form-group password_strength_meter">
+    <div class="col-xs-12 col-sm-12 col-md-6 col-lg-5">
+        <div class="well">
+
+            <div class="form-group">
+                <label class="control-label">{{i18n "users.email.address"}}
```

Comment:
format the label nodes

Reply:
Fixed with https://github.com/wso2/product-is/pull/1373